### ISSUE TITLE
[Bug] InfoWindow anchor not working

### ISSUE THREAD

**Issue by dante-blitz** [visgl/react-google-maps#559]
### Description

I have set an anchor on InfoWindow as per the docs...

```
        <AdvancedMarker
          position={center}
          onClick={toggleInfoWindow}
          ref={markerRef}
        >
          <Pin
            scale={2}
            background="#ff9200"
            borderColor="#000000"
            glyphColor="#000000"
          />
        </AdvancedMarker>
        {infowindowShown && (
          <InfoWindow anchor={marker} onCloseClick={closeInfoWindow}>
            {infoWindowContent}
          </InfoWindow>
        )}
```

The InfoWindow is placed directly on top on the Pin (z axis) and completely obscures it. This same code was previously working. Whenevener next.js hot-reloads, it appears to sort itself out and the InfoWindow is shifted up into the correct position.

### Steps to Reproduce

Load a page that has a gmap on it.

### Environment

"@vis.gl/react-google-maps": "^1.2.0",
"next": "14.2.14",

**Comment by georgekolasa** [comment]
Having this exact issue as well.

**Comment by dante-blitz** [comment]
Still have the issue as well. Haven't yet figured out a workaround that works for me. Manual pixel offset break on page reload or hydration when the anchor starts to work.

**Comment by georgekolasa** [comment]
I'm not using NextJS personally.

**Comment by dante-blitz** [comment]
@mrMetalWood This is happening in dev and production.

**Comment by Tataro** [comment]
I have the same problem.

"next": "14.2.14",
"@vis.gl/react-google-maps": "^1.3.0"

**Comment by mrMetalWood** [comment]
I was able to reproduce the bug and I am working on a fix.

**Comment by mrMetalWood** [comment]
@dante-blitz @georgekolasa @Tataro We published `1.4.0` which should fix this issue. Can you check and see if that is the case for you? Would appreciate a short feedback if the issue is solved.

**Comment by georgekolasa** [comment]
@mrMetalWood LGTM so far, thank you!
